spec/prawn/stamp_spec.rb

```diff
@@ -95,7 +95,7 @@
         next unless obj =~ %r{/Type /Page$}
         # The page object must contain the annotation reference
         # to render a clickable link
-        expect(obj).to match(%r{^/Annots \[\d \d .\]$})
+        expect(obj).to match(%r{^/Annots \[\d+ \d .\]$})
```

Comment:
With the additional object for the XMP metadata stream the object number for the annotation object switched from single digit to double digit (from 9 to 10). This regex only tested for single digit object number. If we make the XMP metadata stream optional, this change can be reverted.